### PR TITLE
Mapbox Mobile Events for iOS 1.0.0-beta

### PR DESCRIPTION
**Pull request by alfwatt**

### CODE REVIEW

File: MapboxMobileEvents/Categories/NSUserDefaults+MMEConfiguration.m [hunk 1]
```diff
@@ -291,19 +291,27 @@ - (NSString *)mme_userAgentString {
...
+        NSMutableSet *loadedMapboxBundleIds = NSMutableSet.new;
+        for (NSBundle *loaded in [NSBundle.allFrameworks arrayByAddingObjectsFromArray:NSBundle.allBundles]) {
+            if (loaded.bundleIdentifier
+             && loaded.bundleIdentifier != NSBundle.mme_mainBundle.bundleIdentifier
+             && [loaded.bundleIdentifier rangeOfString:@"mapbox" options:NSCaseInsensitiveSearch].location != NSNotFound) {
+                [loadedMapboxBundleIds addObject:loaded.bundleIdentifier];
             }
         }
+        
+        // sort the bundleIdentifiers, then use them to build the User-Agent string
```
Comment (rclee) [inline MapboxMobileEvents/Categories/NSUserDefaults+MMEConfiguration.m:314]: 😙👌

File: MapboxMobileEvents/MMEMetricsManager.m [hunk 2]
```diff
@@ -232,10 +221,10 @@ - (NSDictionary *)attributes {
...
         attributes[MMEEventDeviceLon] = @(self.metrics.deviceLon);
     }
-    attributes[MMEEventKeyModel] = [MMEEventsManager sharedManager].commonEventData.model;
-    attributes[MMEEventKeyOperatingSystem] = [MMEEventsManager sharedManager].commonEventData.osVersion;
-    attributes[MMEEventKeyPlatform] = [MMEEventsManager sharedManager].commonEventData.platform;
-    attributes[MMEEventKeyDevice] = [MMEEventsManager sharedManager].commonEventData.device;
+    attributes[MMEEventKeyModel] = MMEEvent.deviceModel;
+    attributes[MMEEventKeyPlatform] = MMEEvent.platformName;
+    attributes[MMEEventKeyOperatingSystem] = MMEEvent.osVersion;
+    attributes[MMEEventKeyDevice] = MMEEvent.deviceModel;
```
Comment (rclee) [inline MapboxMobileEvents/MMEMetricsManager.m]: Should `deviceModel` be repeated here?

Comment (alfwatt) [reply MapboxMobileEvents/MMEMetricsManager.m]: No, no it should not.

/me rustles through diffs

File: MapboxMobileEvents/MMELogger.m [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+}
+
+// MARK: -
+
+- (instancetype)init {
+    if ((self = [super init])) {
+#if DEBUG
+        self.level = MMELogInfo;
+#else
+        self.level = MMEogFatal;
```
[the remark this answers is not in the document]
Comment (alfwatt) [reply MapboxMobileEvents/MMELogger.m]: If only there were a release build target I could validate with...

**Pull request closed without merging**